script.js

```diff
@@ -0,0 +1,50 @@
+class Todo {
+  constructor() {
+    this.id = 0;
+    this.list = [];
+  }
+
+  addItem(title, priority) {
+    const task = {id: this.id++, title, priority};
+    this.list.push(task);
+    return task.id;
+  }
+
+  removeItem(id) {
+    return (this.list.splice(this.list.findIndex(item => item.id === id), 1)).length > 0 ? true : false;
+  }
+
+  getItem(id) {
+    return this.list.find(item => item.id === id);
+  }
+
+  next() {
+    return this.list.reduce((acc, item) => {
+      if(item.priority < acc.priority) {
+        acc = item;
```

Comment:
do not need to reassign acc, just return value which you need, reduce do hard work for you